src/main/java/me/skulduggerry/gameoflife/controller/GenerationController.java

```diff
@@ -26,32 +26,38 @@
 
     @NonNull
     private final ObjectMapper mapper;
+    @NonNull
+    private final GenerationService generationService;
+    @NonNull
+    private final GenerationConverter generationConverter;
+
+    @PostMapping(value = UPLOAD_PATH, consumes = "application/json", produces = "application/json")
+    public String upload(@RequestBody TransferGeneration transferGeneration) {
+        log.debug("Uploaded a valid file");
+        log.debug("Send json:\n" + transferGeneration.toString());
+        generationConverter.toWorkGeneration(transferGeneration);
```

Comment:
Do we need these debug logs in production code?
I am not against it - we should just consiously decide what we want to debug in general.
Also see other debug logs.